Validators/ContactValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FluentValidation;
+using Practice.Models;
+
+namespace Practice.Validators
+{
+    public class ContactValidator : AbstractValidator<Contact>
+    {
+        public ContactValidator()
+        {
+            RuleFor(x => x.Phone)
+                .NotEmpty().WithMessage(ValidationMessages.NotEmpty)
+                .Matches(@"^0[1-9]+$").WithMessage("Phone number not in correct format")
```

Comment:
Please move this regex pattern into a constant class so other places can refer to it. For ex: `Practice.Models.Contact`